src/main/java/org/mitre/synthea/helpers/SimpleCSV.java

```diff
@@ -109,6 +109,31 @@ public static String unparse(List<? extends Map<String, String>> data) throws IO
     return mapper.writer(schemaBuilder.build()).writeValueAsString(data);
   }
 
+  /**
+   * Convert the data in the given List of Maps to a String of CSV data.
+   * Each Map in the List represents one line of the resulting CSV. Uses the keySet from the
+   * first Map to populate the set of columns. This means that the first Map must contain all
+   * the columns desired in the final CSV. The order of the columns is specified by the order
+   * provided by the first Map's keySet, so using an ordered Map implementation
+   * (such as LinkedHashMap) is recommended.
+   *
+   * @param data List of Map data. CSV data read/modified from SimpleCSV.parse(...)
+   * @param separator The separator used in the CSV
+   * @return data formatted as a String containing raw CSV data
+   * @throws IOException on file IO write errors.
+   */
+  public static String unparse(List<? extends Map<String, String>> data, char separator)
+          throws IOException {
+    CsvMapper mapper = new CsvMapper();
+    CsvSchema.Builder schemaBuilder = CsvSchema.builder();
+    schemaBuilder.setUseHeader(true).setColumnSeparator(separator);
+
+    Collection<String> columns = data.get(0).keySet();
+    schemaBuilder.addColumns(columns, ColumnType.STRING);
+
+    return mapper.writer(schemaBuilder.build()).writeValueAsString(data);
+  }
```

Comment:
It wouldn't hurt to add a unit test for this method.

Reply:
Done.